quantization_troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+```python
+core_config = mct.core.CoreConfig(mct.core.QuantizationConfig(shift_negative_activation_correction=True))
+quantizted_model, quantization_info = mct.ptq.keras_post_training_quantization(model, representative_dataset,
+                                                                               core_config=core_config)
+```
+*note:* after activating this flag, you have a few more tweaks to its operation that you can control with the
+`shift_negative_ratio`, `shift_negative_threshold_recalculation` & `shift_negative_params_searc` flags.
+Read all about them in the [quantization config](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/quantization/quantization_config.py#L83) 
+
+
+### Bias Correction
+MCT applies bias correction by default to overcome induced bias shift caused by weights quantization.
+The applied correction is an estimation of the bias shift that is computed based on (among other things) the collected statistical data generated with the representative dataset.
+Therefore, the effect of the bias correction is sensitive to the distribution and size of the provided representative dataset.
+
+**Solution**: Verify the bias correction causes a degradation in accuracy by disabling it:
+```python
+core_config = mct.core.CoreConfig(mct.core.QuantizationConfig(weights_bias_correction=False))
+quantizted_model, quantization_info = mct.ptq.keras_post_training_quantization(model, representative_dataset,
+                                                                               core_config=core_config)
+```
+If you can increase your representative dataset and its distribution, it may restore accuracy.
+If you don't have an option to increase or diversify your representative dataset, disable the bias correction.
+
+
+## Common Issues
+
+### Unbalanced "concatenation"
+The concatenation operation can be a sensitive element in quantized models.
+This is because it involves combining multiple tensors that may have significantly different value ranges, leading to potential inaccuracies in quantization.
+
+For example, the quantization of a tensor that contains values ranging in (-1, 1) together with a tensor that ranges in (-64, 64) is less accurate than the quantization of each tensor separately.
+
+**Solution**:
+You can identify problematic "concatenation" layers using the "network editor" API or by examining the quantization parameters of the input layers to the "concatenation" layer in the quantized model.
+If you discover a concatenation layer with inputs that have notably different threshold values, consider either removing this concatenation, replacing its application to a later point in the model, or balancing its input value ranges by additional an scaling operation.
+
+
+### Pytorch model - common torch.FX errors
+When implementing quantization on a PyTorch model, MCT's initial step involves converting the model into a graph representation using `torch.fx`.
```

Comment:
If here you are writing pytorch as **PyTorch** then make it similar through the entire document